pkg/storage/streams/store.go

```diff
@@ -84,19 +94,31 @@ func (s *streamStore) Put(ctx context.Context, path paths.Path, data io.Reader,
 	var lastSegmentSize int64
 
 	awareLimitReader := EOFAwareReader(data)
+	encKey := sha256.Sum256([]byte(s.key))
```

Comment:
i think the key should just be a []byte already and the hashing should happen somewhere else. maybe there's a helper method that takes a string and bcrypts it into a hash?

Reply:
should I convert it in `NewStreamStore()`?